README.md

```diff
@@ -29,40 +31,58 @@ export default {
   name: 'FormExample',
   components: {
     Form
+  },
+  data() {
+    return {
+      form: {
+        email: '',
+        password: ''
+      }
+    }
+  },
+  methods: {
+    handleSuccessSubmit(values, actions) {
+
+      // Here handle form submit and then call setSubmitting()
+
+      actions.setSubmitting(false)
+    }
   }
 }
 ```
 
-More example how to use it you could find [below](#examples).
+Then in your template:
+
+```html
+<template>
+  <Form
+    @submit="handleSuccessSubmit"
+    :initial-values="form"
+  >
+    <div slot-scope="props">
+      <input v-model="props.values.email" />
+      <input v-model="props.values.password" />
+
+      <button @click="props.handleSubmit">
+        Submit
+      </button>
+    </div>
+  </Form>
+</template>
+```
 
-## Docs
+More example how to use it you could find [below](#examples).
```

Comment:
More examples below, and only H2 with `Example` word, wtf is going here? Let's remove this sentence and in Examples section let's add link to `example-app` package repository, WDYT? @szymonjanota